source/qb64pe.bas

```diff
@@ -12601,8 +12601,8 @@ IF os$ = "WIN" THEN
                     a$ = ReadBufLine$(fh)
```

Comment:
Ok, before messing with line endings I would suggest:
1.) revert this + CHR(10) changes you did, as ReadBufLine$() does stop at EOLs, we rather should tell the buffer to work with Linux/Mac EOLs instead
2.) to do so I think it's time now to replace the reduced utilities/s-buffer version with the full fledged version, grab the simplebuffer.bi/.bm files here https://github.com/RhoSigma-QB64/QB64Library/tree/main/QB64Library/SB-Storage and drop it in the utilities/s-buffer folder just overwriting the old ones
3.) Now place a `ConvBufToLnxMacEol fh` line right after the OpenBuffer%() call above and after the OpenBuffer%() in the second block below

Reply:
Done. 👍